package.json

```diff
@@ -41,6 +41,9 @@
   "engines": {
     "node": ">=0.8"
   },
+  "react-native": {
+    "vm": false
+  },
```

Comment:
If React-Native doesn't support vm anyhow, what does this do?

Reply:
This will tell reacts module bundling not to look for the vm module. Even though the module will not be used during runtime the bundler is still trying to find it.